source/includes/release-notes/fact-release-notes-2-3.rst

```diff
@@ -0,0 +1,25 @@
+|bi| version 2.3 improves enterprise deployment support. The new
+features include:
+
+Improved Schema Management
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+|bi-short| has been simplified to a single component.
+The schema sampling and mapping features of
+:doc:`mongodrdl </reference/mongodrdl>` have added to
```

Comment:
"have added to" reads a little odd. Maybe just 'are included in'

Reply:
Alternatively, just say ":doc:`mongosqld </reference/mongosqld>` now includes the schema mapping functionality previously in :doc:`mongodrl`. `mongodrdl` is no longer necessary in |bi-short| 2.3.